printer_zpl2/wizard/print_record_label.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2016 SYLEAM
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="view_wizard_print_product_label_form" model="ir.ui.view">
+        <field name="name">wizard.print.record.label.form</field>
+        <field name="model">wizard.print.record.label</field>
+        <field name="arch" type="xml">
+            <form string="Print Label">
+                <group>
+                    <field name="printer_id" />
+                    <field
+                        name="label_id"
+                        context="{'default_model_id': active_model_id}"
```

Comment:
Please add this domain="[('model_id', '=', active_model_id)]" to the label_id field because I tested this pull request on my localhost and encountered a problem: the label_id field is always empty, even though I have set a label for the current model. This is happening because the current context doesn't have active_model.